### DISCUSSION TITLE
Moving forward with 0.2 branch - should Python 2.7 support be dropped?

### DISCUSSION THREAD

**Discussion by fredrik-corneliusson** [ranaroussi/yfinance#1146] [Development]
The stable version (0.1 branch) claims to support Python 2.7, but I do not know if it actually does.
Moving forward this would be a hindrance and since Python 2.7 now is over 12 years old and reached end of life nearly 3 years ago.
Due to this I would suggest officially dropping support in upcoming 0.2 releases.
Does anyone have valid reasons not to drop 2.7 support?

In case we drop 2.7 support what would be the oldest 3 version we should support?
Looking at global website statistics (I suspect it is a somewhat lagging indicator) 3.6 is still the most prominent.
https://w3techs.com/technologies/history_details/pl-python/3

**Comment by ValueRaider** [comment]
I found a PyPI stats site with module-specific stats: https://pypistats.org/packages/yfinance

Indicates everyone is using 3.*, and almost everyone on 3.7 minimum.
